content/hi/scalability.md

```diff
@@ -0,0 +1,9 @@
+---
+title: मापनीयता (Scalability)
+status: Feedback Appreciated
+category: बुनियादी बातों
```

Comment:
the category for scalability is 'property' correct? we use `गुण` for that. please replace बुनियादी बातों

Reply:
Alright! Thanks @Garima-Negi

Reply:
@Garima-Negi The changes have been applied! I believe this PR is ready to be merged now

Reply:
awesome ! thank you. Waiting on one more approval